ruby/ql/test/library-tests/ast/ValueText.expected

```diff
@@ -602,7 +602,6 @@
 | modules/modules.rb:55:8:55:30 | "module Foo::Bar again" | module Foo::Bar again |
 | modules/modules.rb:56:17:56:17 | 4 | 4 |
 | modules/toplevel.rb:1:6:1:12 | "world" | world |
-| modules/toplevel.rb:3:12:3:16 | "!!!" | !!! |
```

Comment:
I guess this has disappeared because the `puts` call in `END { puts "!!!" }` is unreachable?

Reply:
yes.